### ISSUE TITLE
dependency issues

### ISSUE THREAD

**Issue by fly** [akerl/targit#17]
```
jchen@hobbes-(master|✔)> gem install targit
ERROR:  While executing gem ... (Gem::DependencyError)
    Unable to resolve dependencies: targit requires octokit (~> 3.7.0); octoauth requires octokit (~> 3.8.0)
jchen@hobbes-(master|✔)> which gem
/usr/local/var/rbenv/shims/gem
jchen@hobbes-(master|✔)> rbenv global
1.9.3-p545
jchen@hobbes-(master|✔)> gem -v
1.8.23.2
```

**Comment by akerl** [comment]
```
❯ gem install targit
Fetching: octokit-3.7.1.gem (100%)
Successfully installed octokit-3.7.1
Fetching: octoauth-1.0.0.gem (100%)
Successfully installed octoauth-1.0.0
Fetching: targit-1.0.0.gem (100%)
Successfully installed targit-1.0.0
3 gems installed
```

Is it possible you have a newer-than-rubygems octoauth installed?

**Comment by akerl** [comment]
Well, replicated on 1.9.3. No idea why this hits 1.9.3 but not whatever newer version

**Comment by fly** [comment]
i'm assuming you're running 2.*? I'll switch to that in the meantime

**Comment by akerl** [comment]
Released 1.0.1 which should resolve this
